index.html

```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
                 Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
                 If you like what you see and have a project you need coded, don’t hesitate and contact me.
             </p>
-            <button class="button1">Start collaboration</button>
+            <button type="button" class="button1">Start collaboration</button>
             <img src="img/image_geometry_2.svg" alt="">
         </section>
+    </div>
+    <section class="gridcontainer works">
+        <h1 class="topic item1">Projects</h1>
+        <div class="item item2">
+            <img src="img/project_1.jpg" alt="laptop">
+            <h1 class="project-name">Project name goes here </h1>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
+            </ul>
+            <button type="button" class="view-project">See this Project &#x2192;</button>
+        </div>
+        <div class="item item3">
+            <img src="img/project_2.jpg" alt="laptop">
+            <h1 class="project-name">Project name goes here </h1>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
+            </ul>
+            <button type="button" class="view-project">See this Project &#x2192;</button>
```

Comment:
- [Optional] Please implement the following button styling: Active, Hover, and Pressed, so the buttons will have these states. Please implement it on all buttons.